issueWork/JS1__issue/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script defer>
+    const numArr = [15,52,33,24,75];
+    let counter = 0;
+    for(let i = 0; i < numArr.length; i++) {
+      numArr[i]+=counter;
+      counter++;
+    }
+    function consoleLoggingAnArray (arr) {
+      let i = 0;
+      while(i < arr.length) {
+
+      if(arr[i] % 2 === 0) {
+        i++;
+        console.log("The number is even.");
```

Comment:
You can improve the utility of your console by doing something like ` The number ${arr[i]} is even`. This way when you look in your console, you can confirm that your logic is in fact accurate. 

Note: for what I am saying, you would need to console first, then increment your counter. I personally prefer to leave the incrementing of the counter for the very end of my code block unless there is a specific need to increment immediately.